data/games/under_night_in_birth_exe_late_st/movespecs/phonon.json

```diff
@@ -0,0 +1,128 @@
+{
+  "name": "Phonon",
+  "movelists": [
+    [
+      "5A",
+      "2A",
+      "5B",
+      "2B",
+      "5C",
+      "5[C]",
+      "2C",
+      "j.A",
+      "j.B",
+      "j.C",
+      "j.[C]"
+    ],
+    [
+      "5BB",
+      "4B",
+      "3C",
+      "j.2B",
+      "j.2[B]",
+      "j.4B"
+    ],
+    [
+      "66B",
+      "66[B]",
+      "66C"
+    ],
+    [
+      {
+        "name": "Impulse",
+        "inputs": [
+          "B+C"
+        ]
+      },
+      {
+        "name": "Impulse (charged)",
+        "inputs": [
+          "[B+C]"
+        ]
+      },
+      {
+        "name": "Throw",
+        "inputs": [
+          "A+D"
+        ]
+      },
+      {
+        "name": "Guard Thrust",
+        "inputs": [
+          "214D"
+        ]
+      },
+      {
+        "name": "Veil Off",
+        "inputs": [
+          "A+B+C+D"
+        ]
+      }
+    ],
+    [
+      {
+        "name": "Affliction",
+        "inputs": [
+          "236A",
+          "236B",
+          "236C"
+        ]
+      },
+      {
+        "name": "Affliction (charged)",
+        "inputs": [
+          "236[A]",
+          "236[B]"
+        ]
+      },
+      {
+        "name": "Affliction (air)",
+        "inputs": [
+          "j.236A",
+          "j.236B",
+          "j.236C"
+        ]
+      },
+      {
+        "name": "Guidance Ascension",
+        "inputs": [
+          "623A",
+          "623B",
+          "623C"
+        ]
+      },
+      {
+        "name": "Rhythmic Satisfaction",
+        "inputs": [
+          "214A",
+          "214B",
+          "214C"
+        ]
+      },
+      {
+        "name": "Restraint",
+        "inputs": [
+          "22A",
+          "22B",
+          "22C"
+        ]
+      }
+    ],
+    [
+      {
+        "name": "Blinding Beatitude",
```

Comment:
Mizuumi says "Binding" instead of "B**l**inding".

Reply:
good catch!